assets/src/components/propertiesPanel/headwayDiagram.tsx

```diff
@@ -35,12 +35,12 @@ const minutes = (seconds: number): string => {
 
 const OtherVehicle = ({ vehicle }: { vehicle: Vehicle }) => {
   const [{ userSettings }, dispatch] = useContext(StateDispatchContext)
-  const { id, viaVariant } = vehicle
+  const { viaVariant } = vehicle
```

Comment:
We should really just do the tech debt task to clean up the headway stuff.

Reply:
I don't really know much about the headway stuff, and I've never heard of anyone using it. We can probably make the case to finally cut it.